### PR TITLE
Broad panel app updates

### PR DESCRIPTION
**Pull request by hanars**
Tweaks and bug fixes to support new PanelApp feature in the Broad's seqr

@tommyli we decided we want to have both the UK and Australia PanelApp lists in seqr, so I updated the loading code to support either instead of one. Let me know if you see any potential issues. Also, I found a couple small bugs you might want to fix up on your end too

### CODE REVIEW

**Comment by tommyli** [comment]
Thanks for those fixes.

There are some issues I see with using both PanelApp AU and PanelApp England together.  You mentioned "either" so does that mean you'll only ever want gene lists from one instance at a time?  Looking at your crontab changes, it looks like you intend to import lists from both instances together?  In either case I see problems.

In the case where you use gene lists from both instances, I think there will be lots of duplication in practice.  I'm not sure how much collaboration there are amongst PanelApp AU and PanelApp England but the sheer fact there are two separate instances tells me the maintenance of these lists goes through different process/reviewers though this is just my perception.  Technically, mixing lists from both instances will not work because of this function:

```python
def _safe_get_locus_list(panel_app_id):
    result = PaLocusList.objects.filter(panel_app_id=panel_app_id)

    return result.first() if result else None
```

In fact, `panel_app_id` is not a good identifier when mixing data from both instances of PanelApp.  Even in the scenario where you're using either one, the `import_all_panels()` logic will not remove previous imported PanelApp gene lists correctly, it'll result in a mixture of gene lists from both PanelApp AU and PanelApp England, depending on which `panel_app_id` clash.

To satisfy what you want, I propose using `PaLocusList.url` as the identifier.  In the case where both instances are used together, then the URL is still unique.  In the case of either, the current logic to import all using URL as identifier will work but additional cleanup method to clear lists from one of the instances is required.  In fact, I'd like to raise a PR to make `PaLocusList.url` as the identifier in the import logic and make this field unique in the model.  What do you think?

Another warning is the performance issue with using gene lists from both instances together.  The UI call to `summary_data/gene_lists` is already fairly slow because of the requirement to search by gene symbol (i.e. in `locust_list_api.locus_lists()`, `include_genes=True`).  I feel this needs to be addressed before using both gene lists together.

**Comment by hanars** [comment]
@tommyli to clarify, we want lists from both at the same time. I think we are okay if there are duplicate lists, we just may want to add an indicator of UK/ Australia to the description to make it easier to differentiate duplicate lists.

I do like the idea of using the URL as the unique identifier, it seems cleaner anyways. Thanks for getting the fix up so quickly!

Based on your performance concerns, I think we use gene lists very differently than you do. We almost never actually go to the page with all the lists, except maybe to make a new list and double check one doesn't exist first. Lists are really used with their integration with search - either for choosing a list to search against, or to annotate the genes in search results to indicate that they are in a list. In either of those cases, only lists that have been explicitly added to a project are considered. As such, creating new lists does not have a performance impact for about 95% of a user's interactions with the lists. 

The only case where the performance would be impacted is on the main lists page (as you noted) and when a user goes to add a new list to a project. These are relatively infrequent activities, as I mentioned. For me locally, with both panel app lists added, it takes about 10 seconds for the main list page to load, and about 7 seconds for the list to load in the "add list" dialog. I consider this to be an acceptable lag time for an infrequent activity, so for our seqr instance I think the value in having access to all the lists outweighs any disadvantages due to performance. While perhaps a performance enhancement would be nice for the future, I am not going to block this feature on it

File: panelapp/models.py [hunk 1]
```diff
@@ -13,13 +13,13 @@ class PaLocusList(models.Model):
         primary_key=True,
     )
 
-    panel_app_id = models.IntegerField(null=False, blank=False, unique=True)
+    url = models.TextField(max_length=2000, null=False, blank=False, unique=True)
+    panel_app_id = models.IntegerField(null=False, blank=False)
```
Comment (ShifaSZ) [inline panelapp/models.py:17]: The `URL` has to incorporate `panel_app_id` to become unique.  How about using unique_together?

**Review by ShifaSZ — CHANGES_REQUESTED** [review]

File: ui/shared/components/panel/variants/Variants.jsx [hunk 2]
```diff
@@ -26,13 +26,13 @@ const StyledVariantRow = styled(({ isCompoundHet, isSV, severity, ...props }) =>
...
+      return '#eaa8a857'
     } else if (severity === 0) {
-      return VARIANT_BACKGROUND_COLORS.amber
+      return '#f5d55c57'
     } else if (severity < 0) {
-      return VARIANT_BACKGROUND_COLORS.green
+      return '#21a92624'
     } else if (isSV) {
-      return VARIANT_BACKGROUND_COLORS.lightBlue
+      return '#f3f8fa'
```
[the remark this answers is not in the document]
Comment (hanars) [reply ui/shared/components/panel/variants/Variants.jsx:35]: as I commented on the other fix PR, this is literally just reverting an accidental change, Theres no reason for this file to change on the current dev build at all. I''m not making optimizations, I'm trying to unbreak the build.

File: panelapp/panelapp_utils.py [hunk 3]
```diff
@@ -47,6 +46,12 @@ def _extract_ensembl_id_from_json(raw_gene_json):
             logger.error('Error occurred when importing gene panel_app_id={}, error={}'.format(panel_app_id, e), user)
 
 
+def delete_all_panels(user, panel_app_api_url):
+    with transaction.atomic():
+        to_delete_qs = SeqrLocusList.objects.filter(palocuslist__url__startswith=panel_app_api_url)
+        SeqrLocusList.bulk_delete(user, queryset=to_delete_qs)
```
[the remark this answers is not in the document]
Comment (hanars) [reply panelapp/panelapp_utils.py:52]: nah this keeps the logging cleaner, we do it in a lot of places

File: panelapp/panelapp_utils.py [hunk 4]
```diff
@@ -7,12 +7,11 @@
 from seqr.utils.gene_utils import parse_locus_list_items
 from seqr.utils.logging_utils import SeqrLogger
 from seqr.views.utils.json_to_orm_utils import update_model_from_json, create_model_from_json
-from settings import PANEL_APP_API_URL
 
 logger = SeqrLogger(__name__)
 
 
-def import_all_panels(user):
+def import_all_panels(user, panel_app_api_url, label=None):
```
[the remark this answers is not in the document]
Comment (hanars) [reply panelapp/panelapp_utils.py:14]: see above, this makes the logging cleaner

File: panelapp/fixtures/panelapp.json [hunk 5]
```diff
@@ -173,5 +213,57 @@
...
+      "last_modified_date": "2018-04-28T00:03:25.341Z",
+      "locus_list": 4,
+      "gene_id": "ENSG00000164120"
+    }
+  },
+  {
+    "model": "panelapp.palocuslist",
+    "pk": 4,
+    "fields": {
+      "panel_app_id": "557",
```
[the remark this answers is not in the document]
Comment (hanars) [reply panelapp/fixtures/panelapp.json:249]: This is part of a separate PR that I already approved, you need to only review my commits. I trust the team who actually knows the data to generate appropriate fixture data

File: panelapp/models.py (hunk 1 above)
Comment (hanars) [reply panelapp/models.py:17]: The URL includes the ID, its the full path

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**